docusaurus.config.js

```diff
@@ -151,51 +151,6 @@ const config = {
       },
       footer: {
         style: 'light',
-        links: [
-          {
-            title: 'Docs',
-            items: [
-              {
-                label: 'Get Started',
-                to: '/docs/tutorials',
-              },
-              {
-                label: 'How-to guides',
-                to: '/docs/guides',
-              },
-            ],
-          },
-          {
-            title: 'Community',
-            items: [
-              {
-                label: 'Twitter',
-                href: 'https://twitter.com/ShifuFramework',
-              },
-              {
-                label: 'Discord',
-                href: 'https://discord.gg/7yXmbQm8ns',
-              },
-              {
-                label: 'Reddit',
-                href: 'https://www.reddit.com/user/ShifuFramework/',
-              },
-            ],
-          },
-          {
-            title: 'More',
-            items: [
-              {
-                label: 'GitHub',
-                href: 'https://github.com/edgenesis/shifu',
-              },
-              {
-                label: 'Download Demo',
-                href: '/disclaimer',
-              },
-            ],
-          },
-        ],
```

Comment:
are we removing all footers?

Reply:
yes，I've already synced with yongli

Reply:
in the future we will move all the links to the navbar